Subject: util_path: Make true/false handling consistent with other functions
From: Jan Hoeppner <jan.hoeppner@de.ibm.com>

Summary:     zpcictl: Add tool to manage PCI devices
Description: Use the zpcictl tool to manage PCI devices on the IBM Z
             platform.  Initial functions include generating firmware
             error logs, resetting PCI devices, and preparing a device
             for further repair actions.
Upstream-ID: 2b92bc4c087fd7a2275ba8fd5608cf3c86cdcc98
Problem-ID:  RAS1703

Upstream-Description:

             util_path: Make true/false handling consistent with other functions

             Signed-off-by: Michael Holzheu <holzheu@linux.vnet.ibm.com>


Signed-off-by: Jan Hoeppner <jan.hoeppner@de.ibm.com>
---
 libutil/util_path.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/libutil/util_path.c
+++ b/libutil/util_path.c
@@ -213,7 +213,7 @@ bool util_path_exists(const char *fmt, .
 	bool rc;
 
 	UTIL_VASPRINTF(&path, fmt, ap);
-	rc = access(path, F_OK) == 0;
+	rc = access(path, F_OK) == 0 ? true : false;
 	free(path);
 	return rc;
 }
